packages/smooth_app/lib/pages/preferences/user_preferences_settings.dart

```diff
@@ -69,8 +69,21 @@ class _ApplicationSettings extends StatelessWidget {
 
     return Column(
       children: <Widget>[
-        UserPreferencesTitle(
-          label: appLocalizations.settings_app_app,
+        Padding(
```

Comment:
I can see this code block getting repeated, maybe you could have made changes to the  `UserPreferencesTitle` if necessary

Reply:
Made changes to **UserPreferencesTitle** and removed the repeated code block